utils/frame/benchmarking-cli/src/shared/mod.rs

```diff
@@ -66,6 +67,53 @@ where
 	s
 }
 
+/// A Handlebars helper to add an underscore after every 3rd character,then format it,
+/// i.e. a separator for large numbers : 123000000 -> 123_000 * WEIGHT_PER_NANOS.
+#[derive(Clone, Copy)]
+pub struct UnderscoreFormatHelper;
+
+impl handlebars::HelperDef for UnderscoreFormatHelper {
+	fn call<'reg: 'rc, 'rc>(
+		&self,
+		h: &handlebars::Helper,
+		_: &handlebars::Handlebars,
+		_: &handlebars::Context,
+		_rc: &mut handlebars::RenderContext,
+		out: &mut dyn handlebars::Output,
+	) -> handlebars::HelperResult {
+		use handlebars::JsonRender;
+		let param = h.param(0).unwrap();
+		let underscore_format_param = underscore_format(param.value().render());
+		out.write(&underscore_format_param)?;
+		Ok(())
+	}
+}
+
+/// Add an underscore after every 3rd character, then format it, i.e. a separator for large numbers:
+/// 123000000 -> 123_000 * WEIGHT_PER_NANOS.
+fn underscore_format<Number>(i: Number) -> String
```

Comment:
The name of this function should be more specific to something with weights, like `picoseconds_to_weight`